src/sonic-config-engine/tests/sample_output/t2-chassis-fe-bgpd.conf

```diff
@@ -37,49 +42,50 @@ router bgp 4000 vrf VnetFE
   exit-address-family
 !!
 !
+! template: bgpd/bgpd.main.conf.j2
 !
 ! bgp multiple-instance
 !
-route-map FROM_BGP_SPEAKER_V4 permit 10
+! BGP configuration
 !
-route-map TO_BGP_SPEAKER_V4 deny 10
+! TSA configuration
 !
 ip prefix-list PL_LoopbackV4 permit 4.0.0.0/32
 !
+ipv6 prefix-list PL_LoopbackV6 permit Invalid ip address None/64
```

Comment:
"Invalid ip address None/64", does this look like correct statement?

Reply:
This is wrong. I'll check what is going on.